python/mxnet/lr_scheduler.py

```diff
@@ -29,8 +30,31 @@ class LRScheduler(object):
     base_lr : float, optional
         The initial learning rate.
     """
-    def __init__(self, base_lr=0.01):
+    def __init__(self, base_lr=0.01, warmup_steps=0, warmup_begin_lr=0, warmup_mode='linear'):
```

Comment:
Do you mind adding documentation for warmup_begin_lr?

Reply:
There was some for the inherited classes, but not for this base abstract class. Anyway, now added for all. Please check.